tendrl/commons/objects/node/atoms/check_node_up/__init__.py

```diff
@@ -0,0 +1,56 @@
+import os
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons.objects import BaseAtom
+from tendrl.commons.objects.node import Node
+
+
+class CheckNodeUp(BaseAtom):
+
+    obj = Node
```

Comment:
you dont need to assign obj, It is done by the commons framework automatically on registering this NS